.github/workflows/generator_generic_slsa3.yml

```diff
@@ -97,6 +97,9 @@ on:
       provenance-name:
         description: "The artifact name of the signed provenance. (A file with the intoto.jsonl extension)."
         value: ${{ jobs.generator.outputs.provenance-name }}
+      provenance-uuid:
```

Comment:
nit: Let's call it `provenance-rekor-uuid`

Reply:
Thank you for the feedback. I have made the requested changes.